types/plugin/quarterOfYear.d.ts

```diff
@@ -16,5 +16,11 @@ declare module 'dayjs' {
     startOf(unit: QUnitType): Dayjs
 
     endOf(unit: QUnitType): Dayjs
+
+    isSame(date: dayjs.ConfigType, unit?: QUnitType): Dayjs
```

Comment:
Would you import { ConfigType } at top of the file, please?

Reply:
Fixed. Thanks